shims.bzl

```diff
@@ -463,6 +463,22 @@ def _default_rust_edition():
 
     return read_config("rust", "default_edition")
 
+def thrift_library(
+        name,
+        thrift_srcs,
+        languages,
+        deps = [],
+        py_base_module = None,
+        rust_deps = [],
+        thrift_rust_options = [],
+        **kwargs):
+    unsupported = []
+    for l in languages:
+        if False:
+            pass
+        else:
+            print("FIXME(buck2-shims-meta): unsupported thrift languages: {}".format(", ".join(unsupported)))
```

Comment:
Shouldn't this do something simpler? Why `if False`? Why log each language if we don't support any of them?